eras/shelley/test-suite/src/Test/Cardano/Ledger/Shelley/Serialisation/EraIndepGenerators.hs

```diff
@@ -120,13 +40,15 @@ genBlock = Block <$> arbitrary <*> (toTxSeq @era <$> arbitrary)
 --
 -- This generator uses 'mkBlock' provide more coherent blocks.
 genCoherentBlock ::
```

Comment:
I think it would be good to move this generator into `testlib` as well. Just make sure it is re-exported from this module

Reply:
Thanks Alexey! Assuming that you mean `TPraos.Arbitrary`, some of the other modules may also need moving to `tpraos`. Not all of the previous closed PR, but just a few, like the `Constants`, besides the `mkBlock` and `mkOCert` functions and the functions they depend on. Is that okay?

Reply:
You know, on a second thought, don't worry about it. I'll do it myself later when I get to the test suite cleanup and organization

Reply:
:+1: Thanks! That sounds good. That way the entire extended task will be done in one sweep, and it might be better to plan together too. :raised_hands: